tyk-headless/values.yaml

```diff
@@ -9,6 +9,10 @@ secrets:
   APISecret: CHANGEME
   OrgID: "1"
 
+# If you don't want to store plaintext secrets in the Helm value file and would rather provide the k8s Secret externally please disable this value
+SecretName:
```

Comment:
I'd suggest to perhaps place it under the `secrets` var and use lowercase first word as convention goes for the Helm values, e.g.:

```.yaml
secrets:
  useSecretName: ""
```